Stop a margin click in the breakpoint margin from performing code folding.
This is because the Komodo breakpoint margin == 2, and it's fold margin == 1.

Index: /Users/toddw/as/komodo-devel/src/scintilla/cocoa/ScintillaView.mm
===================================================================
--- cocoa/ScintillaView.mm	(revision 81864)
+++ cocoa/ScintillaView.mm	(working copy)
@@ -983,12 +1007,13 @@
   {
     case SCN_MARGINCLICK:
     {
-      if (scn->margin == 2)
-      {
-	// Click on the folder margin. Toggle the current line if possible.
-	long line = [self getGeneralProperty: SCI_LINEFROMPOSITION parameter: scn->position];
-	[self setGeneralProperty: SCI_TOGGLEFOLD value: line];
-      }
+      // KOMODO: interferes with Komodo's breakpoint margin - bug 100390.
+      //if (scn->margin == 2)
+      //{
+	// Click on the folder margin. Toggle the current line if possible.
+	//long line = [self getGeneralProperty: SCI_LINEFROMPOSITION parameter: scn->position];
+	//[self setGeneralProperty: SCI_TOGGLEFOLD value: line];
+      //}
       break;
     };
     case SCN_MODIFIED:
